draft-savage-ppm-3phm-mpc.md

```diff
@@ -942,8 +942,8 @@ Instead, each uses Lagrange interpolation to compute a share of `G(r)`.  That
 is:
 
 ~~~ pseudocode
-t_- = lagrange\_interpolate(r, [G_-(0), G_-(1), ..., G_-(2L-2)])
-t_+ = lagrange\_interpolate(r, [G_+(0), G_+(1), ..., G_+(2L-2)])
```

Comment:
The reason I did this is because @martinthomson's converter was converting this into:
```
lagrange<sub>interpolate</sub>
```
due to the use of a really simplistic regular expression =). My "solution" here does not render nicely either though...

Reply:
ahh got it. maybe camel case? `lagrangeInterpolate(...`?

Reply:
why doesn't `total_count` above get convered?

Reply:
> why doesn't `total_count` above get converted?

ahh so two reasons. once because it's in `python` and not `pseudocode`, however that doesn't work here because we want `t_-` and `t_+` to get converted correctly. the second time because only `chars = "-+=()0123456789im"` are used, and `i` makes sense since it's often used as an index, so we can't remove it.

I reverted this, but then also added a small change to @martinthomson's converter to swap out `\_` with `_` after the conversions have been found.

Reply:
@eriktaubeneck made the same change that I did :)  The underscore here is what I went with also.

Reply:
oh it looks like @martinthomson had already fixed this, it just wasn't merged... lol

Reply:
@martinthomson great minds!